packages/eslint-plugin/ROADMAP.md

```diff
@@ -188,11 +196,12 @@ It lists all TSLint rules along side rules from the ESLint ecosystem that are th
 | [`switch-final-break`]              | 🛑  | N/A                                                                                 |
 | [`type-literal-delimiter`]          | ✅  | [`@typescript-eslint/member-delimiter-style`]                                       |
 | [`unnecessary-bind`]                | 🌟  | [`no-extra-bind`][no-extra-bind]                                                    |
-| [`variable-name`]                   | 🌟  | <sup>[2]</sup>                                                                      |
+| [`variable-name`]                   | 🌟  | <sup>[3]</sup>                                                                      |
 | [`whitespace`]                      | 🔌  | Use [Prettier]                                                                      |
 
 <sup>[1]</sup> Recommended config: `["error", { blankLine: "always", prev: "*", next: "return" }]`<br>
-<sup>[2]</sup> [`camelcase`][camelcase], [`no-underscore-dangle`][no-underscore-dangle], [`id-blacklist`][id-blacklist], and/or [`id-match`][id-match]
+<sup>[2]</sup> Doesn't check other control flow statements, such as `break` or `continue`.
+<sup>[3]</sup> [`camelcase`][camelcase], [`no-underscore-dangle`][no-underscore-dangle], [`id-blacklist`][id-blacklist], and/or [`id-match`][id-match]
```

Comment:
ditch all of these.
Instead promote the `naming-convention` rule, which encompasses them all in a more customisable way